crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -596,6 +596,35 @@ type MoveFunction {
 	return: [OpenMoveType!]
 }
 
+type MoveFunctionConnection {
+	"""
+	Information to aid in pagination.
+	"""
+	pageInfo: PageInfo!
+	"""
+	A list of edges.
+	"""
+	edges: [MoveFunctionEdge!]!
+	"""
+	A list of nodes.
+	"""
+	nodes: [MoveFunction!]!
+}
+
+"""
+An edge in a connection.
+"""
+type MoveFunctionEdge {
+	"""
+	The item at the end of the edge
```

Comment:
nit: we call it item here, but otherwise it is defined as a node? Maybe we can directly refer to it as:
`The MoveFunction object at the end of the edge`.